### DISCUSSION TITLE
When title in metadata, can not search and show in search list.

### DISCUSSION THREAD

**Discussion by gledos** [squidfunk/mkdocs-material#3978] [Q&A]
My [website](https://ggame.gledos.science/game/DJMAX_RESPECT_V.html) can not search tile.

![圖片](https://user-images.githubusercontent.com/23720620/171842387-e3a39c6c-7cef-41c3-9382-79443f4a5c37.png)

This page title is "DJMAX RESPECT V".

markdown file is: https://github.com/gledos/ggame/blob/master/docs/game/DJMAX_RESPECT_V.md

---

But can not search and show in search list. may be is [metadata](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#metadata) title not support ?

![圖片](https://user-images.githubusercontent.com/23720620/171843316-b0a9a531-cdf0-4b64-ace7-6acd0952b66d.png)

thank you for your help.

**Comment by squidfunk** [comment]
The metadata title property is supported, e.g. see this page from our documentation which defines "Getting started with Insiders" as a title and it shows up in the search:

![screenshot-squidfunk-github-io-mkdocs-material-insiders-1654256937042](https://user-images.githubusercontent.com/932156/171848029-078ab53c-c640-4692-bea7-9d664664dfb3.png)

If you feel that it doesn't work as expected, please create a minimal reproducible example that showcases the problem.

**Comment by gledos** [comment]
Sorry I forgot to create the minimal example, here is the example: 

website: https://gledos.science/test-metadata/
github repo: https://github.com/gledos/test-metadata

![圖片](https://user-images.githubusercontent.com/23720620/171850536-3aa8062a-472a-4db3-9132-ba443e2d7d6e.png)

**Reply by squidfunk** [reply]
Please create a [new issue](https://github.com/squidfunk/mkdocs-material/issues/new?assignees=&labels=&template=bug.yml) for this, so it's not forgotten.
